src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -342,6 +342,7 @@ export const TitleBar: React.FC<TitleBarProps> = ({
                 css={{marginBottom: 0}}
                 onClick={onClickStartAudio}
                 data-uie-name="do-call"
+                disabled={isReadOnlyConversation}
```

Comment:
This one was already added there https://github.com/wireapp/wire-webapp/pull/16083/files#diff-bdc492056c4abd9bd96ba59134b8a002c8d11abc91d3b555dd1ec1967ae8b8f3R319, but it was missing on medium breakpoint